test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/findModelProperty.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var chai = require('chai');
+var path = require('path');
+var ModelProperty = app.models.ModelProperty;
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example/common/models');
```

Comment:
Is there a before() hook you can do these in so we don't lump them together with the imports?